pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -44,6 +44,9 @@ Base.setproperty!(x::MPtr, f::Union{AbstractString,Int64}, v) = Globals.ASS_REC(
 Base.zero(x::GAPInputType_internal) = Globals.ZERO(x)
 Base.one(x::GAPInputType_internal) = Globals.ONE(x)
 Base.:-(x::GAPInputType_internal) = Globals.AINV(x)
+Base.:-(x::Int64,y::GAP.MPtr) = GAP.julia_to_gap(x)-y
+Base.:*(x::GAP.MPtr,y::String) = x*GAP.julia_to_gap(y)
```

Comment:
Maybe we should try to implement arithmetic by automatically trying to call `julia_to_gap`? Otherwise, this is a very special case, too.

Reply:
https://github.com/oscar-system/GAP.jl/blob/master/pkg/GAPJulia/JuliaInterface/julia/gap.jl#L51 This line plus the `-` below in the loop should actually be sufficient to implement this function. Do you call it on integers larger than 2^60?